pkgs/by-name/fl/flashprog/package.nix

```
@@ -2,12 +2,14 @@
 , installShellFiles
 , lib
 , libftdi1
-, libgpiod_1
+, libgpiod
 , libjaylink
 , libusb1
 , pciutils
 , pkg-config
 , stdenv
+, with_jlink ? true
+, with_gpio ? true
```

Why do we need those options? You didn't provide a justification for them in the thread.

```suggestion
, withJlink ? true
, withGpio ? stdenv.isLinux
```
also we use camleCase

---

camelCase will be deprecated by RFC 169.

---

I don't think it makes sense to try to adhere to an RFC that isn't accepted

---

OK, makes sense.

---

> Why do we need those options?

It is very typical on Nixpkgs.

---

Yeah, I don't see an issue here. They allow people configuring the software. As I just wrote in another comment here, I know cases where NixOS and flashprog is used in embedded solutions. Many times flash sizes are small and you are happy about every single byte you can save. These options make it easier for people to customize flashprog to their use cases. I want to keep that.

---

I think this can be marked as resolved. I can't do that though.

---

Though, as long as the RFC is not approved yet, we should go with camel case. Please change that.

---

I changed it to camelCase.